pkg/updater/resultstore/resultstore.go

```diff
@@ -294,6 +337,30 @@ func includeStatus(tg *configpb.TestGroup, sar *singleActionResult) bool {
 	return true
 }
 
+// testResult is a convenient representation of resultstore Test proto
+// only one of those fields are set at any time for a testResult instance
+type testResult struct {
+	suiteProto *resultstorepb.TestSuite
+	caseProto  *resultstorepb.TestCase
+}
+
+// properties return the recursive list of properties for a particular testResult
+func (t *testResult) properties() []*resultstorepb.Property {
+	var properties []*resultstorepb.Property
+	for _, p := range t.suiteProto.GetProperties() {
+		properties = append(properties, p)
+	}
+	for _, p := range t.caseProto.GetProperties() {
+		properties = append(properties, p)
+	}
+
+	for _, t := range t.suiteProto.GetTests() {
+		newTestResult := &testResult{t.GetTestSuite(), t.GetTestCase()}
```

Comment:
nit: I don't think the `&` does anything here?

Reply:
i think you're right - you can provide both a pointer and variable to a pointer receiver method